statistical_method_for_oxide_growth/statistical_method_for_oxide_growth.md

```diff
@@ -0,0 +1,24 @@
+# Problem statement
+We experience very poor quality of the oxide grown on the silicon wafers on both dry and wet thermal oxidation.  
+We observe the results in cases:
+
+1. Grown oxide does not provide a good dopant stop. We read N-type semiconductor in places that should have not been doped, because we grew 500nm thick oxide.
```

Comment:
we read -> we observe

Reply:
How about `we read N-type semiconductor from the hot point probe test`? Which is what I wanted to write originally, but unconsciously ended up writing this crap.

Reply:
```diff
- from the hot point ...
+ during the hot point ...
```
?

Reply:
> ```diff
> - from the hot point ...
> + during the hot point ...
> ```
> 
> ?

yeah, i think ``during`` sounds much better.